### PR TITLE
Script to generate Java and Python client

### LINKED ISSUE #778: [FEATURES] Provide Java client
**What feature you'd like to add:**

Fluid Java Client. 

**Why is this feature needed:**

Many distributed bigdata and scheduling system (like ODPS Fuxi Dag2) is written by Java. When the fluid api  changes, I hope fluid community can update and maintain client code.

**related project**

OpenKruise has provided Java Client(https://github.com/openkruise/client-java). These java code is generated by openapi and swagger. you could talk with @FillZpp （酒祝 in Ali） about it.

### PR DESCRIPTION
**Pull request by Rui-Tang**
### Ⅰ. Describe what this PR does

This PR  automatically convert Go structs to OpenAPI to JSONSchema to SDK of multiple languages

### Ⅱ. Does this pull request fix one issue?

fixes #778 

### Ⅴ. Special notes for reviews
We can move SDK to seperate repositoris like OpenKruise java client (https://github.com/openkruise/client-java)

### CODE REVIEW

**Comment by allenhaozi** [comment]
It's better to have a separate repo as the SDK's repo @cheyang @RongGu

**Comment by cheyang** [comment]
Yeah, we already have the repo: https://github.com/fluid-cloudnative/fluid-client-java

**Comment by xieydd** [comment]
So, we will close the pr and seperate the python and java sdk to independent repo?

**Comment by Rui-Tang** [comment]
> So, we will close the pr and seperate the python and java sdk to independent repo?

No, I will clean this PR, remove python and java client. Scripts to generate client code will remain.

File: hack/sdk/main.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 kubeflow.org.
```
Comment (cheyang) [inline hack/sdk/main.go:2]: Please update license header. Thanks.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: hack/sdk/main.go (hunk 1 above)
Comment (Rui-Tang) [reply hack/sdk/main.go]: ok

File: docs/zh/dev/multiple-client-support.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+## 如何使用其他语言（非Go语言）客户端
+
+Fluid 通过 kube-openapi 与 swagger-codegen 可以支持多种语言，包括 Java、Python等。
+
```
Comment (cheyang) [inline docs/zh/dev/multiple-client-support.md:4]: Does it require Java 1.8+ ?

**Review by cheyang — APPROVED** [review]

**Pull request merged**